agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -244,6 +245,11 @@ public void customize(AutoConfigurationCustomizer autoConfiguration) {
             (builder, otelConfig) ->
                 configureMetrics(metricFilters, builder, telemetryClient, configuration));
 
+    AiContextCustomizerHolder.setInstance(
```

Comment:
does this impact connectionString polling for azure spring cloud? 
it's a one time setup.. i believe we don't support connectionstring or rolename override for rp-integration.. if yes, we're good.

Reply:
ya, we don't support polling for connection string (or role name) overrides